src/pages/LandingPage.tsx

```
@@ -0,0 +1,75 @@
+        </Link>
+      </div>
+
+      {/* GitHub Link */}
+      <a 
+        href="https://github.com/tumigisla/zenpool"
+        target="_blank"
+        rel="noopener noreferrer"
+        className="absolute bottom-6 sm:bottom-8 text-white/20 hover:text-white/50 transition-colors duration-300"
+        title="View on GitHub"
+      >
+        <svg className="w-5 h-5 sm:w-6 sm:h-6" fill="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)